qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -74,6 +83,12 @@ def __init__(self, coupling_map, heuristic="basic", seed=None, fake_run=False):
             seed (int): random seed used to tie-break among candidate swaps.
             fake_run (bool): if true, it only pretend to do routing, i.e., no
                 swap is effectively added.
+            max_iterations_without_progress (int): The maximum number of swaps that will be
+                attempted if no progress seems to be being made (i.e. without additional gates being
+                routed).  Once this value is exceeded, swaps will be greedily inserted (ignoring the
+                heuristic) to force a new gate to be routed, and the pass will then continue
+                normally.  If not specified, this defaults to being proportional to the number of
+                qubits in the given :class:`.DAGCircuit`.
```

Comment:
While this is useful explanation I wonder how user actionable this is. Like if I didn't have the context of the bug and the code I'm not sure I'd understand what this is saying or when/why I would want to tweak this argument. I'm not sure if I have a concrete suggestion here, I was thinking maybe mention something about it being per layer but I'm not sure that would help.

That being said thinking about it more, while I typically thing values like this should be tweakable I'm not sure a user would ever want to change this as they likely won't have sufficient context to pick something better for a particular application. Maybe we should just make this internal only and not expose it with a kwarg on the constructor?

Reply:
Yeah, I think we can get rid of this.  I added it originally because I needed it for my first idea for tests (call `Operator` on a smaller circuit), but then when I came on the idea of just using Aer to simulate the keys I didn't need it any more, because Aer can very quickly simulate the infinite-loop circuit (it's got barely any entanglement).

Removed (and the now-unnecessary test too) in 1d5d8bda5.